src/settings.c

```diff
@@ -355,6 +355,49 @@ void load_settings(char *cmdline_config_path)
                 "Window corner radius"
         );
 
+        settings.progress_bar_height = option_get_int(
+                "global",
+                "progress_bar_height", "-progress_bar_height", defaults.progress_bar_height,
+                "Height of the progress bar"
+        );
+
+        settings.progress_bar_min_width = option_get_int(
+                "global",
+                "progress_bar_min_width", "-progress_bar_min_width", defaults.progress_bar_min_width,
+                "Minimum width of the progress bar"
+        );
+
+        settings.progress_bar_max_width = option_get_int(
+                "global",
+                "progress_bar_max_width", "-progress_bar_max_width", defaults.progress_bar_max_width,
+                "Maximum width of the progress bar"
+        );
+
+        settings.progress_bar_frame_width = option_get_int(
+                "global",
+                "progress_bar_frame_width", "-progress_bar_frame_width", defaults.progress_bar_frame_width,
+                "Frame width of the progress bar"
+        );
+
+        settings.progress_bar = option_get_bool(
+                "global",
+                "progress_bar", "-progress_bar", false,
```

Comment:
You can set this to `true` as well, the goal is to have this and `dunstrc` in sync at some point not make it diverge more :)

Reply:
Oh I forgot that one. I changed it now